app/src/main/java/net/squanchy/settings/view/SettingsHeaderLayout.java

```diff
@@ -85,7 +85,10 @@ private void updateUserPhotoFrom(UserInfo userInfo) {
         Optional<String> photoUrl = photoUrlFor(userInfo);
         if (photoUrl.isPresent()) {
             imageLoader.load(photoUrl.get())
+                    .error(R.drawable.ic_no_avatar)
                     .into(userPhotoView);
+        } else {
```

Comment:
that's already 💯 but maybe we can do something like `photoUrl.or(R.drawable.ic_no_avatar)` to remove the if/else? 😄 

Same applies to the other places

Reply:
Yeah maybe in the future, the problem is that those urls are strings and having an extension method on them that turns them into ints and then getting rid of type safety in the image loader to support the scenario seems... worse XD

Reply:
Actually disregard the imageloader bit; we still have to figure out a way to create a decent API for this that is actually feasible. I cannot think of a way off the top of my head